src/app/auth/account/account.component.html

```diff
@@ -1 +1,174 @@
+                        value="datasets"
+                        [checked]="isAccountViewTypeDatasets"
+                        (click)="onSelectDatasetsTab()"
+                    >
+                        <mat-icon class="text-muted fs-4">book</mat-icon>
+                        <span class="mx-2">Datasets</span>
+                        <span
+                            title="1"
+                            data-view-component="true"
+                            class="counter"
+                            >6</span
+                        >
+                    </mat-button-toggle>
+                    <mat-button-toggle
+                        data-test-id="account-organizations-tab"
+                        value="organizations"
+                        [checked]="isAccountViewTypeOrganizations"
+                        (click)="onSelectOrganizationsTab()"
+                    >
+                        <mat-icon class="text-muted fs-4">people</mat-icon>
+                        <span class="mx-2">Organizations </span>
+                        <span
+                            title="1"
+                            data-view-component="true"
+                            class="counter"
+                            >2</span
+                        >
+                    </mat-button-toggle>
+
+                    <mat-button-toggle
+                        data-test-id="account-inbox-tab"
+                        value="inbox"
+                        [checked]="isAccountViewTypeInbox"
+                        (click)="onSelectInboxTab()"
+                    >
+                        <mat-icon class="text-muted fs-4">inbox</mat-icon>
+                        <span class="mx-2">Inbox</span>
+                        <span
+                            title="1"
+                            data-view-component="true"
+                            class="counter"
+                            >2</span
+                        >
+                    </mat-button-toggle>
+                    <mat-button-toggle
+                        data-test-id="account-stars-tab"
+                        value="Stars"
+                        [checked]="isAccountViewTypeStars"
+                        (click)="onSelectStarsTab()"
+                    >
+                        <mat-icon class="text-muted fs-4">star_border</mat-icon>
+                        <span class="mx-2">Stars</span>
+                        <span
+                            title="1"
+                            data-view-component="true"
+                            class="counter"
+                            >2</span
+                        >
+                    </mat-button-toggle>
+                </mat-button-toggle-group>
+            </div>
+            <ng-container *ngIf="isAccountViewTypeOverview">
+                <div>Overview</div>
+            </ng-container>
+            <ng-container *ngIf="isAccountViewTypeDatasets">
```

Comment:
Potenttially, the functionality will grow. So, let's move the "datasets" tab into a separate child component. It's okay to keep the other not-implemented tabs as is for now, but as soon as there will be some functionality there, those should also be in the separate components. Similarly to dataset view tabs

Reply:
The pagination control also needs to move to this new component